### ISSUE TITLE
upgrading delta.chat within anbox

### ISSUE THREAD

**Issue by hpk42** [deltachat/deltachat-android#188]
i successfully installed "com.b44t.messenger_433.apk" with "adb install" into anbox.  Now i want to upgrade to 453 -- how do i do that?

**Comment by r10s** [comment]
i have not tested it with andbox, but i would try

    adb install -r com.b44t.messenger_453.apk

**Comment by Ampli-fier** [comment]
I also did not manage to get Anbox running ... but there should be a browser from where you can download and open (=install) the (any) app.

**Comment by haeckser** [comment]
if you have installed delta chat by apk there is no update  except creating a backup, deinstallation  and install it with the new apk (and import the backup).

if you would like to have delta chat up to date you should add F-Droid  via apk to Anbox and make updates with F-Droid.

What is your base system - Ubuntu 17.x ? If so, you can install Anbox in snap format as described on homepage of anbox 
$  sudo snap install --classic anbox-installer && anbox-installer

**Comment by Ampli-fier** [comment]
I tried different versions and flavours of Ubuntu (incl. Mint, LUbuntu), but all of them in _**VirtualBox**_.

The result was always the same, anbox would not start:

`libEGL warning: DRI2: failed to authenticate
[ 2017-10-20 12:39:47] [Renderer.cpp:190@initialize] Failed: Missing egl_image related extension(s)
Segmentation fault
`

This is reported to Anbox' issues several times, but no solution seems to be available.

I gave up and installed Android directly in VirtualBox, using an image from the [Android-x86 Project](http://www.android-x86.org/).
